static/js/base/ga.js

```diff
@@ -72,7 +72,9 @@ if (typeof dataLayer !== "undefined") {
       return;
     }
 
-    for (let key in events) {
+    let key;
```

Comment:
I can't find a rule that allows this - it seems to be a recommended pattern

Reply:
I think is a eslint + babel-eslint bug: https://github.com/eslint/eslint/issues/12117

Maybe we should update all eslint related deps.